package.json

```diff
@@ -5,8 +5,10 @@
   "dependencies": {
     "@capacitor/android": "2.3.0",
     "@capacitor/core": "2.3.0",
+    "@capacitor/ios": "^2.2.0",
+    "@ionic-native/barcode-scanner": "^5.27.0",
```

Comment:
Bitte hinzugekommene Packages bzw. entfernte Packages ohne '^' und dann entsprechend in die ThirdPartyNotice - Datei mit aufnehmen oder entsprechend vermerk machen.